adobe/spectrum-web-components
# build: use older lerna so @swc-react packages can be published

## Description
Rollback to `lerna@6.x` so we can continue to use the existing workflow for publishing `@swc-react` packages.

## How has this been tested?
    1. Load the branch locally
    2. Run `yarn`
    3. Run `yarn build:react` (builds react wrappers, which are `.gitignore`d)
    4. Run `yarn lerna list`
    5. See that `@swc-react` packages are included

## Types of changes

## Checklist
